asapdiscovery-alchemy/asapdiscovery/alchemy/cli.py

```diff
@@ -165,9 +165,8 @@ def submit(network: str, organization: str, campaign: str, project: str):
         project: The name of the project this network should be submitted under.
     """
     from alchemiscale import Scope
-
-    from .schema.fec import FreeEnergyCalculationNetwork
-    from .simulation.utils import AlchemiscaleHelper
```

Comment:
Rationale for switching back to absolute imports here? Not exactly opposed, but usually prefer relative imports in library code.

Reply:
This was just my personal preference, but I don't have a strong opinion on either. I just noticed than in the other subpackages we were using absolute imports for the most part. I think it's okay to mix them to make things more readable.